challenges/objects-arrays.js

```diff
@@ -43,23 +68,33 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for(let i = 0; i<graduates.length; i++){
+  universities.push(graduates[i].university);
+  universities.sort();
```

Comment:
you can sort after the for loop, no point in sorting it multiple times, just sort it at the end